content/concept-amendments.md

```diff
@@ -111,7 +113,8 @@ The following is a comprehensive list of all known amendments and their status o
 
 | Name                            | Introduced | Enabled |
```

Comment:
Should the column name be changed to "Status"  since not all amendments are going to be enabled?